video/rest/compositions/get-composition-media-file/get-composition-media-file.8.x.java

```diff
@@ -0,0 +1,41 @@
+// Install the Java helper library from twilio.com/docs/java/install
```

Comment:
When I copy this and run it, I get a few errors about some weird special characters:
```./Main.java:17: error: illegal character: '\u200b'

^
./Main.java:23: error: illegal character: '\u200b'

^
./Main.java:31: error: illegal character: '\u200b'

^
./Main.java:33: error: illegal character: '\u200b'

^
./Main.java:37: error: illegal character: '\u200b'
```

I guess somehow some empty whitespace characters popped into the file?

Here's an example command you can run to get rid of them: https://superuser.com/questions/207207/how-can-i-delete-u200b-zero-width-space-using-sed

Or a suggestion about how to do it with VSCode https://stackoverflow.com/questions/32013346/zero-width-space-character-in-string-literals-detect-and-or-prevent

Reply:
I think this file is also missing some of the required imports. The one you sent me in Slack works though (after removing the special characters).